msf_updates/src/position_pose_msf/position_pose_sensormanager.h

```diff
@@ -186,8 +186,15 @@ class PositionPoseSensorManager : public msf_core::MSF_SensorManagerROS<
     Eigen::Quaterniond yawq(cos(yawinit / 2), 0, 0, sin(yawinit / 2));
     yawq.normalize();
 
-    q = yawq;
-    q_wv = (q * q_ic * q_vc.conjugate()).conjugate();
+    q_wv = yawq.conjugate();
+
+    if (q_vc.w() == 1) {  // If there is no pose measurement, only apply q_wv.
```

Comment:
should I add a flag in the sensor handler to track if a measurement has arrived?